src/state/campaigns/hooks.ts

```diff
@@ -29,6 +31,20 @@ export function useSelectedCampaignLeaderboardPageNumberManager() {
   )
 }
 
+export function useSelectedCampaignLuckyWinnerPageNumber(): [number, (page: number) => void] {
+  const page = useSelector((state: AppState) => state.campaigns.selectedCampaignLuckyWinnersPageNumber)
+  const dispatch = useDispatch()
+
+  const setPage = useCallback(
+    (newPageNumber: number) => {
+      dispatch(setSelectedCampaignLuckyWinnersPageNumber(newPageNumber))
+    },
+    [dispatch],
+  )
+
+  return useMemo(() => [page, setPage], [page, setPage])
```

Comment:
I think useMemo is not quite needed here. we can remove it

Reply:
It does need indeed. `return [page, setPage]` creates new instance of array regards of value of `page` and `setPage` are same as before

Reply:
But why do you need to memo them? I see he spreaded them eventually 
```js
const [currentPageLuckywinner, setCurrentPageLuckywinner] = useSelectedCampaignLuckyWinnerPageNumber()
```

Reply:
ah, after looking again,  I also see that it was not necessary. I just copied and edited the name so I didn't notice.

sorry. :D
I updated.

Reply:
🤝

Reply:
![image](https://user-images.githubusercontent.com/33005392/190545878-a7bc2c0c-e574-4583-b7c6-18dcd7bf871d.png)

Reply:
![image](https://user-images.githubusercontent.com/23650671/190552767-bbd5db73-d6e2-4bc1-80b5-1bda5bc8b025.png)